devices/xiaomi-tissot/default.nix

```diff
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (config.mobile.device) name;
+in {
+  mobile.device.name = "xiaomi-tissot";
+  mobile.device.info = (lib.importJSON ../postmarketOS-devices.json).${name} // {
```

Comment:
See #34; this is going to be removed...

The data we need can be found using unpackbootimg, or well, could be added to the attrset here directly, from the json data. Relying on the internal scheme from a third party is not ideal in the end.